src/app/pages/my-request-detail/my-request-detail.component.ts

```diff
@@ -257,18 +224,18 @@ export class MyRequestDetailComponent implements OnInit {
     }
   }
 
-  removeSelected() {
-    const itemList = this.requestElements.filter((item) => item.isSelected);
-    this.okCancelItemList(itemList)
+  removeOneOrMoreDataElements(items: DataElementSearchResult[]) {
```

Comment:
The refactoring of the remove element code into a common function is generally good. It could probably even be made into a `private` function 👍

Reply:
made it private